Agda.cabal

```diff
@@ -201,7 +201,7 @@ flag enable-cluster-counting
     False, then the --count-clusters flag triggers an error message.)
 
 flag optimise-heavily
```

Comment:
While you are at it could you duplicate this as `optimize-heavily` (AE spelling)?

Reply:
is there a graceful way to duplicate in the cabal file, without literally duplicating flag declaration and action?

Reply:
You can at least avoid some duplication by prefixing the action with an or condition, but otherwise I don't think there is

Reply:
Actually that was a bad idea of mine, because the following thing will be confusing `cabal -foptimize-heavily -f-optimise-heavily`: If someone turned on the flag before you the AE way, you cannot turn it off the BE way... :-)